src/ui/components/map/providers/mapprovider.js

```diff
@@ -88,6 +95,14 @@ export default class MapProvider {
     };
   }
 
+  shouldHideMap (mapData, resultsContext) {
```

Comment:
What if someone had not set noResults.visible but had _showEmptyMap as true? Is that handled correctly here? In that case, if there's a query that doesn't return actual results, wouldn't they expect to see a Map?

Reply:
I think in that case we want to hide the map. The spec says if noResults is not specified, "don't show" for no results.

Reply:
i.e. we don't need to fall back to past showEmptyMap behavior

Reply:
I'd check with Rose about the behavior in this case. I'm not sure if the spec meant "Default if noResults not specified" or "Default if noResults not specified and showEmptyMap not specifed"

Reply:
yup just posted in slack

Reply:
Ok apparently I didn't misread things, 
| so noResults.visible is true, show the map on no results. if noResults.displayAllResults is also true, show the pins on the map

Reply:
if only noResults.visible is not specified it should still hide the map for no results